### ISSUE TITLE
[docker-based prerelease] --level argument not working?

### ISSUE THREAD

**Issue by 130s** [ros-infrastructure/ros_buildfarm#285]
In the following, I thought I excluded downstream testing by `--level 0`, with which I expect 0 packages to be tested in `catkin_workspace_overlay`, but I see this many pkgs being tested (and thus the prerelease test seems never-ending on my machine). Am I wrong?

```
$ generate_prerelease_script.py   https://raw.githubusercontent.com/ros-infrastructure/ros_buildfarm_config/production/index.yaml   indigo default ubuntu trusty amd64   moveit_core   --level 0   --output-dir ./
$ ./prerelease.sh 
```
```
$ ls catkin_workspace/src/
moveit_core  moveit_msgs  moveit_ros
$ ls catkin_workspace_overlay/src/
abb_irb2400_moveit_config          fetch_moveit_config            pepper_moveit_config
abb_irb6640_moveit_config          force_torque_sensor_calib      play_motion
atlas_moveit_config                grasping_msgs                  pr2_arm_kinematics
atlas_v3_moveit_config             hironx_moveit_config           pr2eus_moveit
baxter_moveit_config               husky_ur5_moveit_config        pr2_moveit_config
baxter_sim_kinematics              industrial_trajectory_filters  pr2_moveit_tutorials
carl_moveit                        iri_wam_moveit_config          pr2_teleop_general
clam_moveit_config                 jaco_moveit_config             r2_moveit_generated
cob_collision_monitor              jsk_ik_server                  robot_calibration
cob_grasp_generation               jsk_interactive_marker         romeo_moveit_actions
cob_kinematics                     jsk_pcl_ros                    romeo_moveit_config
cob_lookat_action                  jsk_pcl_ros_utils              simple_grasping
cob_moveit_config                  katana                         sr_robot_msgs
cob_obstacle_distance              mico_moveit_config             staro_moveit_config
cob_obstacle_distance_moveit       moveit_commander               tiago_moveit_config
cob_pick_place_action              moveit_core                    turtlebot_arm_block_manipulation
fanuc_lrmate200ic5h_moveit_config  moveit_full                    turtlebot_arm_moveit_config
fanuc_lrmate200ic5l_moveit_config  moveit_planners_ompl           turtlebot_arm_moveit_demos
fanuc_lrmate200ic_moveit_config    moveit_python                  ur10_moveit_config
fanuc_m10ia_moveit_config          moveit_setup_assistant         ur3_moveit_config
fanuc_m16ib20_moveit_config        moveit_sim_controller          ur5_moveit_config
fanuc_m20ia10l_moveit_config       moveit_simple_grasps           ur_kinematics
fanuc_m20ia_moveit_config          moveit_visual_tools            vs060
fanuc_m430ia2f_moveit_config       nao_moveit_config              vs060_moveit_config
fanuc_m430ia2p_moveit_config       nextage_moveit_config
```

**Comment by dirk-thomas** [comment]
If I run your prerelease the script outputs the following:

```
Clone release repositories for overlay workspace

Underlay workspace contains 1 packages:
- moveit_core
Overlay packages based on dependency level: 0
Overlay packages based on whitelisted package names: 0
Overlay workspace will contain 0 packages:
```

Please provide more information to reproduce the problem.

**Comment by 130s** [comment]
I figured out that the folders are remains from the previous prerelease test -- I guess I misunderstood that `prerelease.sh` would have removed remaining folders from the previous test when it actually doesn't.

When I manually removed `/tmp/prerelease_job` and re-ran the prerelease script, I confirm downstream test didn't occur. So this ticket is invalid; `--level 0` seems to be working as expected.

However, assuming there's no use of those remaining folders from the previous test once the user finished the test and the relevant work (e.g. reproducing errors), I think it's better for `prerelease.sh` to remove those folders when it detects them, to avoid confusion. I opened #286 to address this..
